bin/utils/common.sh

```diff
@@ -44,6 +44,15 @@ log_message() {
   fi
 }
 
+# print message to stdout and also write message to log
+print_and_log_message() {
+  message=$1
+  echo "${message}"
+  if [[ -n "${LOG_FILE}" ]] && [[ -w "${LOG_FILE}" ]]; then
+    echo "${message}" >> $LOG_FILE
+  fi
```

Comment:
This code looks like it can be refactored with log_message

Reply:
The `log_message` has chance to write to stdout instead of log file. So if we call `print_message` then `log_message`, we have chance to write twice to stdout. If we can change log_message to never write to stdout, we should be ok. Just I don't know what consequences could be. Do you think it's ok to change `log_message` by removing this:
```
  else
    echo ${message}
```

Reply:
I see, without changing the log_message to take an extra paramter maybe it is best to keep as is then